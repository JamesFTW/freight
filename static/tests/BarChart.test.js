import React from 'react';
import renderer from 'react-test-renderer';
import BarChart from '../components/BarChart.jsx';


test('BarChart Snapshot', () => {
  const component = renderer.create(
    <BarChart />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
