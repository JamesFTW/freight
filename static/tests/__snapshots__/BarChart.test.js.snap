// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarChart Snapshot 1`] = `
<figure
  className=" barchart"
  height={undefined}
  width={undefined}
>
  <span
    className="max-y"
  >
    10
  </span>
  <span
    className="min-y"
  >
    0
  </span>
</figure>
`;
